### ISSUE TITLE
PrimoBO: "Check that string exists"

### ISSUE THREAD

**Issue by LincolnUniLTL** [primousers/primo-ve-norm#15]
Primo back office has the "Check that string exists" transformation built in out of the box. Can this be replicated in XML?

**Comment by mwan-work** [comment]
I had a scenario where I needed to display the location of an item. The information was in the metadata, but it was inconsistent in how it was display on different records.

i.e. I needed to know if an item was in the 'Main library' or not, but the metadata from the source has different variations: "Main Library" , "University of X - Main Library" , "MAIN" etc. The only consistency was the word "Main".

So I created this rule that looks for the word 'Main', regardless of case sensitivity. If it exists, then it would put in "Main Library" as the value for a local field. This made it more consistent on our Primo.

```
rule "Main"
when

  exist "//*[local-name()='physloc'][contains(translate(text(),'ABCDEFGHIJKLMNOPQRSTUVWXYZ','abcdefghijklmnopqrstuvwxyz'),'main')]

then

  set "Main Library" in "discovery"."local14"

end

```
